js/packages/teams-ai/src/Application.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { strict as assert } from 'assert';
```

Comment:
should we put test related code to `test` folder?

Reply:
That's what Steve originally had, but then he decided to change it and removed the `test` folder. I'm not 100% sure why but I know it was intentional.

Reply:
I think for now we can merge this, we can always move tests to a `test` folder later on and its pretty standard practice in typescript projects for test files to live in the same directory as their corresponding source files.